egs/chime1/s5/local/chime1_prepare_data.sh

```diff
@@ -40,9 +40,8 @@ scp="$data/train/wav.scp"
 rm -f "$scp"
 for sid in `seq 34`; do
   sid2=`printf "s%02d" $sid`
-  tab=$'\t'
   ls -1 $wav_train/id$sid/*.wav \
-    | sed "s/\(.*\)\/\(.*\).wav/${sid2}_\2${tab}\1\/\2.wav/" \
+    | perl -ape "s/(.*)\/(.*).wav/${sid2}_\2\t\1\/\2.wav/;" \
```

Comment:
You can't assume that these advanced features are compatible; you'll have to test it.  I suggest to just figure out a single line that you think should probably match, and try both versions.  However, in this particular case  you could probably leave the sed there.  It's the newline which we really have a problem with, the tab can be dealt with using the existing solution.

Reply:
yep that's what I did regarding tests. Do you still want to switch back to our old solution using sed?